README.md

```diff
@@ -64,15 +64,26 @@ Also check out [@ProjectAntrea](https://twitter.com/ProjectAntrea) on Twitter!
 
 ## Features
 
-Antrea currently supports the following features:
-* IPv4 overlay network for a Kubernetes cluster. Geneve, VXLAN, GRE, or STT can
-be used as the encapsulation protocol.
-* [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies)
-implementation.
-* [Octant](https://github.com/vmware-tanzu/octant) UI plugin for monitoring
-Antrea components, which publish runtime information as
-[CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
-* [IPsec encyption](/docs/ipsec-tunnel.md) of GRE tunnel traffic.
+* **Kubernetes-native**: Antrea follows best practices to extend the Kubernetes
+  APIs and provide familiar abstractions to users, while also leveraging
+  Kubernetes libraries in its own implementation.
+* **Run everywhere**: Multiple traffic modes (with or without overlay) are
+  supported to ensure that Antrea can run in private clouds, public clouds and
+  at the edge.
+* **Windows Node support**: Thanks to the portability of Open vSwitch, Antrea
+  can use the same data plane implementation on both Linux and Windows
+  Kubernetes Nodes.
+* **Comprehensive policy model**: Antrea provides a comprehensive network policy
+  model, which builds upon Kubernetes Network Policies with new features such as
+  policy tiering, rule priorities and cluster-level policies.
+* **Troubleshooting and monitoring tools**: Antrea comes with a CLI tool
+  (antctl) and an Octant UI plugin, it exposes Prometheus metrics and supports
```

Comment:
Worth  to mention Traceflow and policy diagnostics (I meant query API/CLI, and state/flow correlation in different ways/features, but probably need not to list them one by one).

Reply:
I will add some information but indeed I don't want to start listing specific features here